chapter_notation/index.md

```diff
@@ -1,85 +1,94 @@
+* $x_{ij}$, $x_{i,j}$,$[\mathbf{X}]_{ij}$, $[\mathbf{X}]_{i,j}$: the element of matrix $\mathbf{X}$ at row $i$ and column $j$.
 
 
 
 
 ## Set Theory
 
 
-* $\mathcal{X}$: A set
-* $\mathbb{Z}$: The set of integers
-* $\mathbb{Z}^+$: The set of positive integers
-* $\mathbb{R}$: The set of real numbers
-* $\mathbb{R}^n$: The set of $n$-dimensional vectors of real numbers
+* $\mathcal{X}$: a set
+* $\mathbb{Z}$: the set of integers
+* $\mathbb{Z}^+$: the set of positive integers
+* $\mathbb{R}$: the set of real numbers
+* $\mathbb{R}^n$: the set of $n$-dimensional vectors of real numbers
 * $\mathbb{R}^{a\times b}$: The set of matrices of real numbers with $a$ rows and $b$ columns
-* $|\mathcal{X}|$: Cardinality (number of elements) of set $\mathcal{X}$
-* $\mathcal{A}\cup\mathcal{B}$: Union of sets $\mathcal{A}$ and $\mathcal{B}$
-* $\mathcal{A}\cap\mathcal{B}$: Intersection of sets $\mathcal{A}$ and $\mathcal{B}$
-* $\mathcal{A}\setminus\mathcal{B}$: Subtraction of set $\mathcal{B}$ from set $\mathcal{A}$
+* $|\mathcal{X}|$: cardinality (number of elements) of set $\mathcal{X}$
+* $\mathcal{A}\cup\mathcal{B}$: union of sets $\mathcal{A}$ and $\mathcal{B}$
+* $\mathcal{A}\cap\mathcal{B}$: intersection of sets $\mathcal{A}$ and $\mathcal{B}$
+* $\mathcal{A}\setminus\mathcal{B}$: set subtraction of $\mathcal{B}$ from $\mathcal{A}$ (contains only those elements of $\mathcal{A} that do not belong to $\mathcal{B}$)
 
 
 ## Functions and Operators
 
 
-* $f(\cdot)$: A function
-* $\log(\cdot)$: The natural logarithm
-* $\exp(\cdot)$: The exponential function
-* $\mathbf{(\cdot)}^\top$: Transpose of a vector or a matrix
-* $\mathbf{X}^{-1}$: Inverse of matrix $\mathbf{X}$
+* $f(\cdot)$: a function
+* $\log(\cdot)$: the natural logarithm (base $e$)
+* $\log_2(\cdot)$: logarithm with base $2$
+* $\exp(\cdot)$: the exponential function
+* $\mathbf{1}(\cdot)$: the indicator function, evaluates to $1$ if the boolean argument is true and $0$ otherwise
+* $\mathbf{1}_{\mathcal{X}}(z)$: the set-membership indicator function, evaluates to $1$ if the element $z$ belongs to the set $\mathcal{X}$ and $0$ otherwise
+* $\mathbf{(\cdot)}^\top$: transpose of a vector or a matrix
+* $\mathbf{X}^{-1}$: inverse of matrix $\mathbf{X}$
 * $\odot$: Hadamard (elementwise) product
-* $[\cdot, \cdot]$: Concatenation
-* $\lvert \mathcal{X} \rvert$: Cardinality of set $\mathcal{X}$
+* $[\cdot, \cdot]$: concatenation
 * $\|\cdot\|_p$: $L_p$ norm
 * $\|\cdot\|$: $L_2$ norm
-* $\langle \mathbf{x}, \mathbf{y} \rangle$: Dot product of vectors $\mathbf{x}$ and $\mathbf{y}$
-* $\sum$: Series addition
-* $\prod$: Series multiplication
-* $\stackrel{\mathrm{def}}{=}$: Definition
+* $\langle \mathbf{x}, \mathbf{y} \rangle$: dot product of vectors $\mathbf{x}$ and $\mathbf{y}$
+* $\sum_i=1^n$: summation over collection with indices ranging from $1$ to $n$
+* $\prod$: product over collection with indices ranging from $1$ to $n$
```

Comment:
First, currently there is no $n$ in the $\prod$ notation.
Second, in the book we often use a summation over a set (e.g., for all tokens in a vocabulary), so we may not limit the case to sum from 1 to n. That's why we made the definition as general "addition/multiplication"

Reply:
fixed, made it more generic